src/lib/y2storage/proposal/space_maker_prospects/partition_prospect.rb

```diff
@@ -52,6 +52,16 @@ def linux_in_disk?
           @linux_in_disk = analyzer.linux_partitions(disk_name).any?
         end
 
+        # Whether there was a Windows system in the same disk of the target
+        # partition (in the original devicegraph).
+        #
+        # @return [Boolean]
+        def windows_in_disk?
```

Comment:
The same than above, why not use `if` w/o `nil?` instead `unless` or directly a memoization?

Reply:
Same than above.